README.md

```diff
@@ -36,7 +36,7 @@ $ cd alfresco
 Type following command to produce Docker Compose template in current folder (like `alfresco`) when using Linux or MacOS:
 
 ```
-$ docker run -it -v $(pwd):/generated angelborroy/alfresco-installer
+$ docker run -it -v $(pwd)/generated angelborroy/alfresco-installer
```

Comment:
This makes no sense to me, syntax expected is

```
docker run -v local_folder:container_folder
```

Without ":" there is no mapping between local and container folder